### DISCUSSION TITLE
Welcome to DIGEdit Discussions!

### DISCUSSION THREAD

**Discussion by kingToolbox** [kingToolbox/WindEdit#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Let's do our best to create a high performance editor.

**Comment by Vivybestwaifu** [comment]
Hi kingToolbox,

When I am using the software as a terminal, I wonder how I can edit a file in the filer with the Editor Mode? That means, instead of opening a new intance of WindEdit (when it is set as the default editor), use the editor mode of the same session to edit the file. Thank you in advance for your time.

Best regards,

**Comment by DreamCreat** [comment]
Hello @kingToolbox 

I have a huge request to implement this idea. It consists in making an unconstrained string. That is, 2,147,648,467 characters fit on one line. I would like to fit 1,099,511,627,776 characters. Why so many? I just really want to implement my ideas in real life, but not one text editor developer can work with one line normally.

Please make my dream come true


У меня огромная просьба реализовать такую идею. Она заключается в том , что сделать безограничений строку. То есть в одной строке помещается 2,147,648,467 символов. Мне бы хотелось бы поместить 1,099,511,627,776 символов. Почему именно так много? Просто я очень хочу реализовать свои идеи в реали , но не один разработчик текстовых редакторов не предоставрляет нормально работать с одной строкой. 

Пожайлуста исполни мою мечту

**Reply by aliyoung** [reply]
Try EmEditor

**Comment by emperor-mato** [comment]
Hey @kingToolbox ,

I only recently found out about your excellent tools so I have not yet tested them all, except WindEdit and it takes the crown across all editors I tried by long shot. I am using very old desktop PC running AMD Athlon II X2 240 so every minor action can render pc almost unusable. I work on a hobby project, it includes curating and minor transformations to datasets usually in .csv format. Until now I used Notepad++, which was taking forever to load datasets above 50-100mb and lock you out of using computer because of lag/freezing. WindEdit loads the biggest one I currently have on hand(1,6GB) in a few seconds and it is barely noticeable that something is happening - I can freely use computer without noticing  lack of performance. I came here to leave a BIG THANK YOU with hopes you will have chance to read it. Your project extremely improved my situation and work on hobby project without frustration. Thank you
